src/components/Tooltip/index.scss

```diff
@@ -58,20 +58,42 @@
     cursor: default;
     display: none;
     white-space: pre;
+    width: fit-content;
+    height: fit-content;
+
     &.position-right {
       transform: translateX(-8%);
     }
-    &::after {
-      content: "";
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-style: solid;
-      border-width: 7px 5px 0 5px;
-      border-color: $tooltip-backgroundColor transparent transparent transparent;
-      top: 98%;
-      left: 50%;
-      transform: translateX(-50%);
+
+    &.top {
```

Comment:
how about
`
&::after {
        content: "";
        position: absolute;
        width: 0;
        height: 0;
        border-style: solid;
        transform: translateX(-50%);
      }
`
then declare `&::after` for `&.top` and `&.right`?

Reply:
updated, thank you